src/test/features/dashboard/routes/index.ts

```diff
@@ -29,7 +47,7 @@ describe('Dashboard page', () => {
 
       it('should return 500 and render error page when cannot retrieve claims', async () => {
         draftStoreServiceMock.resolveFind('claim')
-        claimStoreServiceMock.rejectRetrieveByClaimantId('HTTP error')
+        rejectRetrieveByClaimantId('HTTP error')
```

Comment:
I am guessing the reason for `import * as claimStoreServiceMock from 'test/http-mocks/claim-store'` is to clearly say its a mock, not sure. I think we do this everywhere, should we maybe continue that?

Reply:
👍 I think it makes it less legible but probably more explicit, have changed that